x-pack/legacy/plugins/siem/public/components/timeline/timeline.tsx

```diff
@@ -178,6 +184,7 @@ export const TimelineComponent: React.FC<Props> = ({
             id={id}
             indexPattern={indexPattern}
             dataProviders={dataProviders}
+            filterManager={filterManager}
```

Comment:
Oh I see why it is not possible to use the context, but I think it might be the time to move this query to a hook so we do not deal with this problem all the time. We can do a follow-up PR to clean it up.